tests/eas/load_tracking.py

```diff
@@ -537,6 +540,406 @@ def test_load_avg_behaviour(self, experiment, tasks):
+        window = self._get_stable_window(sw_df, tasks_desc)
+
+        msg = 'Saw util {} on cpu {}, expected {} during phase {}'
+        for phase in phases:
+            cpus = self._get_cpus(tasks_desc, phase)
+            # Get the mean utilization per CPU
+            util_mean = self._get_util_mean(window[phase][0], window[phase][1],
+                                            util_df, cpus)
+            # Get the expected utilization per CPU
+            expected = self._get_util_expected(tasks_desc, cpus, phase)
+
+            # Check that the expected utilization value and the measured one
+            # match for each CPU.
+            for cpu in cpus:
+                self.assertAlmostEqual(util_mean[cpu], expected[cpu],
+                                       delta=self.allowed_util_margin,
+                                       msg=msg.format(util_mean[cpu], cpu,
+                                                      expected[cpu], phase))
+
+class OneTaskCPUMigrationTest(_CPUMigrationBase):
+    """
+    Goal
+    ====
+    Checks that the utilization is correctly updated for CPU when a task is
+    migrated from a CPU to another.
+
+    Detailed Description
+    ====================
+    Three tasks are created. Each task defined two phases as follow (the CPU
+    is the one corresponding for the hikey960 it may be another CPU on
+    another platform):
+
+              Phase 0         Phase 1
+            CPU | Util      CPU | Util
+    Task 0:   4 | 25%         5 | 25%
+    Task 1:   4 | 10%         4 | 10%
+    Task 2:   5 | 5%          5 | 5%
+
+    Expected Behaviour
+    ==================
+    The utilization of a CPU is sampled during both phases and should be
+    observed as follow:
+
+            Phase 0         Phase 1
+            Util            Util
+    CPU 4:   35%             10%
+    CPU 5:   5%              30%
+    """
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
+        # Get the 2 CPUs with the highest capacities
+        cpus = sorted(test_env.calibration(),
+                         key=test_env.calibration().get)[:2]
+        tasks = []
+        # tasks creation
+        task0_phase0 = Periodic(duty_cycle_pct=25, duration_s=1, period_ms=16,
+                                cpus=cpus[0])
+        task0_phase1 = Periodic(duty_cycle_pct=25, duration_s=1, period_ms=16,
+                                cpus=cpus[1])
+        tasks.append(task0_phase0 + task0_phase1)
+
+        task1_phase = Periodic(duty_cycle_pct=10, duration_s=1, period_ms=16,
+                               cpus=cpus[0])
+        tasks.append(task1_phase + task1_phase)
+
+        task2_phase = Periodic(duty_cycle_pct=5, duration_s=1, period_ms=16,
+                               cpus=cpus[1])
+        tasks.append(task2_phase + task2_phase)
+
+        json_file = cls.generate_json(test_env, tasks, 'cpu_migration0')
+        wload = cls.get_wload(json_file)
```

Comment:
So, with the above comment... this will be a single call to `get_wload`

Reply:
Done